Assignments/Trello/Python/User.py

```diff
@@ -0,0 +1,8 @@
+class User:
+
+    def __init__(self,id):
+        self.id=id
```

Comment:
Are you able to even compile the code succesfully ?. Can you upload a working code ?